### PR TITLE
Create react app

### PR DESCRIPTION
**Pull request by dubistdu**

### CODE REVIEW

File: client/jlk-racing/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "jlk-racing",
+  "version": "0.1.0",
```
Comment (kaxcode) [inline client/jlk-racing/package.json]: make version 1.0.0 to match API

**Comment by dubistdu** [comment]
@Kenia 
oh soot....yea I made a mistake. Should have been developer branch not master. Will take care of it

File: client/jlk-racing/package.json (hunk 1 above)
Comment (kaxcode) [reply client/jlk-racing/package.json]: yes

**Comment by lizthrilla** [comment]
i just switched it to developer for you @dubistdu

**Pull request merged**